src/compiler/crystal/tools/doc/generator.cr

```diff
@@ -1,7 +1,13 @@
 class Crystal::Doc::Generator
+  class Options
```

Comment:
If we keep this, it could be a struct (using record macro) to reduce memory impact. Naming could also be improved, maybe `GeneratorConfig`?